handlers/get_network_handler.py

```diff
@@ -0,0 +1,25 @@
+import json
+from typing import Union
+
+from aws_lambda_context import LambdaContext
+
+from usecases.get_network import GetNetwork
```

Comment:
Every `get_network` or `GetNetwork` in this file are now called `get_node` and `GetNode`

Reply:
The name of the file could also be changed to `get_node_handler`

Reply:
changed